### PR TITLE
fix: Congratulations popup after buying an NFT is not availiable anymore?

### LINKED ISSUE #7336: Congratulations popup after buying an NFT is not availiable anymore?
I bought an NFT, and the only popup I got was this one:
NFT bought on ahk - tested only on production

Screenshot 2023-09-24 at 13 16 01

Where is our nice, pretty Congratulations popup created by exez we had till now?

### PR DESCRIPTION
**Pull request by hassnian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7336

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; 

CleanShot 2023-10-06 at 11 33 26@2x

### <samp>🤖 Generated by Copilot at e4327e1</samp>

This pull request enhances the user feedback when buying an item from the gallery. It adds a `MessageNotify` component that shows a custom success message with the item names, and updates the `Buy` component and the `useTransaction` composable to support this feature. It also adjusts some styles and types to improve the UI and code quality.

### <samp>🤖 Generated by Copilot at e4327e1</samp>

> _`Buy` component moved_
> _Shows `MessageNotify` now_
> _Autumn leaves of code_

### CODE REVIEW

**Comment by hassnian** [comment]
@roiLeo I'm substituting this popup and not showing the other notification , I think it does not make any sense to have both with the same info

File: components/buy/Buy.vue [hunk 1]
```diff
@@ -76,7 +80,7 @@ const handleBuy = async (nfts: TokenToBuy | TokenToBuy[]) => {
       interaction: ShoppingActions.BUY,
       nfts,
       urlPrefix: urlPrefix.value,
-      successMessage: $i18n.t('mint.successNewNfts'),
+      successMessage: null,
```
Comment (preschian) [inline components/buy/Buy.vue:83]: What if we pass another parameter to trigger a bigger notification component instead of `null`?

Comment (hassnian) [reply components/buy/Buy.vue:83]: sure

File: layouts/explore-layout.vue [hunk 2]
```diff
@@ -31,11 +31,18 @@
...
       <ListingCartModal />
     </template>
+
+    <Buy @success="handleBuy" />
+
+    <MessageNotify
+      v-if="message"
+      :title="message.title"
+      :subtitle="message.subtitle"
+      @close="message = null" />
```
Comment (roiLeo) [inline layouts/explore-layout.vue:45]: Do we really need to put some logic in layout? it feels bad for me

any other option? (maybe put it in another component)

**Comment by prury** [comment]
it works now! tested on snek
button border a bit odd
image

**Comment by hassnian** [comment]
@prury now it's squared :D

CleanShot 2023-10-06 at 11 18 41@2x

it was not missing there , but i spotted the old messages that are still there , will remove that 
CleanShot 2023-10-06 at 11 20 54@2x

try again anywhere else where you can open the buy modal , cart , explore page , profile , etc

**Review by preschian — APPROVED** [review]

**Comment by prury** [comment]
> @prury now it's squared :D
> 
> CleanShot 2023-10-06 at 11 18 41@2x
> 
> it was not missing there , but i spotted the old messages that are still there , will remove that (done) CleanShot 2023-10-06 at 11 20 54@2x
> 
> feel free try again anywhere else where you can open the buy modal , cart , explore pages, collection , profile , etc , it's the same code so there should be no issue

oh, nice catch!

**Comment by prury** [comment]
Perfect, ty @hassnian 
image

**Review by prury — APPROVED** [review]

**Review by exezbcz — APPROVED** [review]

File: components/shared/modals/ModalWrapper.vue [hunk 3]
```diff
@@ -32,31 +34,39 @@
...
 </script>
 
 <style scoped lang="scss">
 .modal-wrapper-button__right {
   float: right;
 }
+.is-squared {
+  width: 36px;
+  height: 36px;
+}
```
Comment (roiLeo) [inline components/shared/modals/ModalWrapper.vue:71]: maybe icon is the problem, can't we do withtout fixed size?

**Review by roiLeo — APPROVED** [review]
✅ otherwise code lgtm

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 3.82 USD/DOT ~ 7.853 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x6385575d192e71250d9ecbc2e3ce47f9f22626b917bbc8985d112c4b90cacc1b](https://polkadot.subscan.io/extrinsic/0x6385575d192e71250d9ecbc2e3ce47f9f22626b917bbc8985d112c4b90cacc1b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**